src/examples/java/com/amazonaws/crypto/examples/FileStreamingDefaults.java

```diff
@@ -0,0 +1,144 @@
+        final File decryptedFile = new File(sourcePlaintextFile.getPath() + ".decrypted");
+        encryptedFile.deleteOnExit();
+        decryptedFile.deleteOnExit();
+
+        // Prepare your encryption context
+        // https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/concepts.html#encryption-context
+        final Map<String, String> encryptionContext = new HashMap<>();
+        encryptionContext.put("encryption", "context");
+        encryptionContext.put("is not", "secret");
+        encryptionContext.put("but adds", "useful metadata");
+        encryptionContext.put("that can help you", "be confident that");
+        encryptionContext.put("the data you are handling", "is what you think it is");
+
+        // Create the keyring that determines how your data keys are protected.
+        final Keyring keyring = StandardKeyrings.awsKms(awsKmsCmk);
+
+        // Create the encrypting input stream with the keyring and encryption context.
+        // Because the file might be too large to load into memory,
+        // we stream the data, instead of loading it all at once.
+        try (final AwsCryptoInputStream encryptingStream = awsEncryptionSdk.createEncryptingInputStream(
+                CreateEncryptingInputStreamRequest.builder()
+                        .keyring(keyring)
+                        .encryptionContext(encryptionContext)
+                        .inputStream(new FileInputStream(sourcePlaintextFile)).build())) {
+
+            // Copy the encrypted data into the encrypted file.
+            try (FileOutputStream out = new FileOutputStream(encryptedFile)) {
+                IOUtils.copy(encryptingStream, out);
+            }
+        }
+
+        // Demonstrate that the ciphertext and plaintext are different.
+        assert !compareFiles(sourcePlaintextFile, encryptedFile);
+
+        // Create the decrypting input stream with the keyring.
+        try (final AwsCryptoInputStream decryptingStream = awsEncryptionSdk.createDecryptingInputStream(
+                CreateDecryptingInputStreamRequest.builder()
+                        .keyring(keyring)
+                        .inputStream(new FileInputStream(encryptedFile)).build())) {
+
+            // Check the encryption context before we start decrypting.
+            //
+            // Verify that the encryption context used in the decrypt operation includes
+            // the encryption context that you specified when encrypting.
+            // The AWS Encryption SDK can add pairs, so don't require an exact match.
+            //
+            // In production, always use a meaningful encryption context.
+            final AwsCryptoResult<AwsCryptoInputStream> decryptResult = decryptingStream.getAwsCryptoResult();
+            encryptionContext.forEach((k, v) -> {
+                assert v.equals(decryptResult.getEncryptionContext().get(k));
+            });
+
+            // Copy the plaintext data to a file
```

Comment:
"Copy" to me implies that we're just taking plaintext from one place and putting it in another place, where what we are actually doing is taking an encrypted message, decrypting it, and putting the resulting plaintext somewhere.

We should also be consistent:
```suggestion
            // Now that we are more confident that we will decrypt the right message,
            // we can start decrypting.
```

Reply:
Updated the wording